app/templates/index.html

```diff
@@ -41,5 +41,38 @@
         </div>
         <h1>{{ title }}</h1>
     </div>
+
+    <div>
+        <style>
+            h2 {
+                font-size: 40px;
+                color: #1C539F;
+                font-size: 40px;
+                text-align: center;
+                text-underline-offset: 5px;
+            }
+        </style>
+        <h2><u>Work Experience</u></h2>
+    </div>
+
+    {% for experience in experiences %}
```

Comment:
I propose to add a tab system, similar to the one used in #19 so we could filter the information for each one of us, what do folks think?

Reply:
Done! How's it look to you?